src/identity/components/Name.tsx

```diff
@@ -46,11 +53,9 @@ export function Name({
         className={cn(text.headline, className)}
         {...props}
       >
-        {name ?? getSlicedAddress(contextAddress ?? address)}
+        {name ?? formattedAddress}
```

Comment:
If `<Name>` returns either a `name` or a `formattedAddress`, why the `getName` utility doesn't do the same?

A component is just a visualizzation of an utility. So should `getName` return `formattedAddress`?

Reply:
That's a good point that the component should reflect the utility.

The utility fetches the `ENS` name for a given address. If the this lookup returns `null`, I'd expect the function to also return `null`. 

I would not expect `isName` to return something other than `null` or the `ENS` name. 

If we want to have `isName` return a formatted address, then we also must pass the `isSliced` boolean to the `isName` function, which I feel would be strange.